### ISSUE TITLE
Allow users to call /jobs/add

### ISSUE THREAD

**Issue by kofalt** [scitran/core#296]
From `api/jobs/handlers.py`:

```python
	def add(self):
		"""
		Add a job to the queue.
		"""

		# TODO: Check each input container for R, check dest container for RW
		if not self.superuser_request:
			self.abort(403, 'Request requires superuser')
```

@rentzso, would you mind lending me a hand?

Based on our conversation yesterday I was able to add simple permission checking for the new endpoint `/container/id/jobs`, but I am not sure how to implement the `TODO` mentioned in the code above.

**Comment by rentzso** [comment]
Sure. Assigning this to myself.

**Comment by kofalt** [comment]
Cool.

Further on after that snippet, you'll see that the `inputs` variable is a `string -> FileReference` map, and the `destination` variable is a `ContainerReference`. My assumption is the auth check will have to move there, right before the `job = Job(` line.

Oh, and both of those types can be found in `api/dao/containerutil.py`.

**Comment by elisapoquette** [comment]
Dependency for Adhoc gear prototype - short term, comment it out so @dpuccetti can start work

**Comment by kofalt** [comment]
Workaround placed in branch for scitran/core#301.

**Comment by kofalt** [comment]
This was resolved in scitran/core#301 :+1:
